scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram.py

```diff
@@ -151,7 +152,11 @@ def beam_search_eval(
     model.cfg.decoding.beam = cfg.decoding
 
     # Update model's decoding strategy
-    model.change_decoding_strategy(model.cfg.decoding)
+    if isinstance(model, EncDecHybridRNNTCTCModel):
```

Comment:
Why we assume the Hybrid models are supposed to be used with ctc decoder? Doesn't this script work with RNNT models?